test/export/data/subdivisions_test.rb

```diff
@@ -0,0 +1,33 @@
+# encoding: utf-8
+
+require File.expand_path(File.join(File.dirname(__FILE__) + '/../../test_helper'))
+
+class TestCldrDataSubdivisions < Test::Unit::TestCase
+  test 'subdivisions for Canada/Quebec in :ja' do
+    subdivisions = Cldr::Export::Data::Subdivisions.new(:ja)[:subdivisions]
+
+    assert_equal('ケベック州', subdivisions[:caqc])
+  end
+
+  test 'subdivisions for a non an unsupported locale :zz' do
+    subdivisions = Cldr::Export::Data::Subdivisions.new(:zz)
+
+    assert_empty(subdivisions)
+  end
+
+  test 'subdivisions locales are a subset of main locales' do
+    root = File.expand_path('./vendor/cldr/common')
+
+    main_locales = Dir["#{root}/main/*.xml"].map { |path| path =~ /([\w_-]+)\.xml/ && $1 }
+    subdivisions_locales = Dir["#{root}/subdivisions/*.xml"].map { |path| path =~ /([\w_-]+)\.xml/ && $1 }
+
+    assert_empty(subdivisions_locales - main_locales)
+  end
+
+ #
+ #   test "extract subdivisions for #{locale}" do
```

Comment:
Do we still need this?

Reply:
Just following an existing pattern where we have this kind of tests commented out 🤷‍♂️. Honestly, I think we should just have something that runs all exports. It takes like 5 mins.